azimuth_capi/operator.py

```diff
@@ -27,42 +28,40 @@
 
 logger = logging.getLogger(__name__)
 
-
 # Constants for target API versions
 CLUSTER_API_VERSION = "cluster.x-k8s.io/v1beta1"
 CLUSTER_API_CONTROLPLANE_VERSION = f"controlplane.{CLUSTER_API_VERSION}"
 AZIMUTH_SCHEDULING_VERSION = "scheduling.azimuth.stackhpc.com/v1alpha1"
 
-
-# Create an easykube client from the environment
-from pydantic.json import pydantic_encoder
-ekclient = (
-    Configuration
-        .from_environment(json_encoder = pydantic_encoder)
-        .async_client(default_field_manager = settings.easykube_field_manager)
-)
-
-
-# Create a Helm client to target the underlying cluster
-helm_client = HelmClient(
-    default_timeout = settings.helm_client.default_timeout,
-    executable = settings.helm_client.executable,
-    history_max_revisions = settings.helm_client.history_max_revisions,
-    insecure_skip_tls_verify = settings.helm_client.insecure_skip_tls_verify,
-    unpack_directory = settings.helm_client.unpack_directory
-)
-
-
-# Create a registry of custom resources and populate it from the models module
-registry = CustomResourceRegistry(settings.api_group, settings.crd_categories)
-registry.discover_models(models)
+# these are initialised during startup
+ekclient = None
+helm_client = None
+registry = None
```

Comment:
This change is a bit risky... lets see how that goes. Normally I would add a method here, but that is a very invasive change.